docs/doxygen/demos/demos_subpages.txt

```diff
@@ -1,8 +1,8 @@
 /**
 @page mqtt_demo_basic_tls Basic TLS Demo
-@brief Uses OpenSSL to establish a TLS connection with server authentication, and demonstrates QoS 2 implementation.
+@brief Uses an OpenSSL-based transport interface implementation to establish a TLS connection with server authentication, and demonstrates QoS 2 implementation.
```

Comment:
Suggest keeping the brief small and concise. The detail of the OpenSSL based transport interface implementation can be mentioned in the paragraph.
```suggestion
@brief Demo of an MQTT application that uses QoS 2 level of communication with broker over a TLS connection for server and client authentication.```